### PR TITLE
✨Source ExchangeRates: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #29276: Exchange Rates Api
https://docs.airbyte.com/integrations/sources/exchangeratesapi

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Exchange Rates to Low-Code CDK*
Closes #29276

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Exchange Rates</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Test results: All passed state
image

**Comment by marcosmarxm** [comment]
Hello 👋 , right now Airbyte's integration tests are failing for this connector and I'm going to work during the week to fix them. Due a high number of contributions we have a backlog and your contribution will be review next week. We ask to you share the integration test from your side whatever possible and if you don't have credentials to run tests to get in touch with @RealChrisSean as soon as possible. Also some common missing steps contributors does migrating to low code are (and you can double check to make sure you implemented this): 
* adding paginator
* update the documentation changelog + metadata file and dockerfile version
* if the connector has `start_date` parameter you must implement incremental syncs

**Comment by btkcodedev** [comment]
Checks:
- No incremental support ✅ 
- No pagination support ✅ 
- Updated version in docker file, docs, and metadata :)  ✅

File: airbyte-integrations/connectors/source-exchange-rates/setup.py [hunk 1]
```diff
@@ -5,19 +5,24 @@
...
-
-TEST_REQUIREMENTS = ["pytest~=6.1", "pytest-mock~=3.6.1", "requests-mock"]
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.1",
+]
 
+TEST_REQUIREMENTS = [
+    "pytest~=6.2",
+    "pytest-mock~=3.6.1",
+    "connector-acceptance-test",
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-exchange-rates/setup.py:15]:
```suggestion
```

**Review by sh4sh — APPROVED** [review]

**Comment by sh4sh** [comment]
Thanks for your contribution!

**Comment by btkcodedev** [comment]
@sh4sh Thank you for your continuous efforts

**Pull request merged**